pyproject.toml

```diff
@@ -1,4 +1,7 @@
 
 [tool.taskipy.tasks]
+setup = "pre-commit install"
```

Comment:
Running `task setup` installs our commit hooks so that it can run our pre-commit hook setup.